src/se/vidstige/jadb/HostConnectToRemoteTcpDevice.java

```diff
@@ -0,0 +1,76 @@
+package se.vidstige.jadb;
+
+import se.vidstige.jadb.entities.TcpAddressEntity;
+
+import java.io.IOException;
+
+class HostConnectToRemoteTcpDevice {
+    private final Transport transport;
+    private final ResponseValidator responseValidator;
+
+    HostConnectToRemoteTcpDevice(Transport transport) {
+        this.transport = transport;
+        this.responseValidator = new ResponseValidatorImp();
+    }
+
+    //Visible for testing
+    HostConnectToRemoteTcpDevice(Transport transport, ResponseValidator responseValidator) {
+        this.transport = transport;
+        this.responseValidator = responseValidator;
+    }
+
+    TcpAddressEntity connect(TcpAddressEntity tcpAddressEntity)
+            throws IOException, JadbException, ConnectionToRemoteDeviceException {
+        transport.send(String.format("host:connect:%s:%d", tcpAddressEntity.getHost(), tcpAddressEntity.getPort()));
```

Comment:
The string "host:", etc is otherwise mostly found in the `JadbConnection` class. I think this feature should also go in that class to keep the design consistent.